shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -2250,4 +2252,11 @@ object GlutenConfig {
       .doc("If enabled, gluten will convert the viewfs path to hdfs path in scala side")
       .booleanConf
       .createWithDefault(false)
+
+  val VELOX_FROM_JSON_FUNC_ENABLED =
+    buildConf("spark.gluten.sql.velox.fromJson.enabled")
+      .internal()
+      .doc("When true, offload from_json function to Velox.")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
The existing `spark.gluten.expression.blacklist` can be used to disable some functions. Seems no need to introduce a dedicated one.